### ISSUE TITLE
Command line interaction

### ISSUE THREAD

**Issue by OmegaLambda1998** [th-ch/youtube-music#998]
It would be very helpful to be able to interact with youtube-music via the command line. For instance:
```
youtube-music play # Start playing whatever the current song is
youtube-music paused # Stop playing whatever the current song is
youtube-music toggle # Swap between playing and pausing
youtube-music next # Go to the next song
youtube-music prev # Go to the previous song
youtube-music info # Information on currently playing song, title, artist, length, position, etc...
```

This will be particularly useful for DM's like I3 and Sway, as it means we can control youtube-music via keystrokes, for instance `win+space` might run `youtube-music toggle` and allow me to start / stop the song. I'd also be able to integrate youtube-music into waybar / i3-bar, allowing me to see the current playing song in my header bar.

**Comment by Araxeus** [comment]
Will be possible after #946 which adds a protocol handler

**Comment by Zo-Bro-23** [comment]
Also should be possible through MPRIS right?

**Comment by Araxeus** [comment]
duplicate of #484 #653 #776 #739 #704
